about.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose?styles=44178" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>About</title>
+</head>
+
+<body>
+    <header>
+        <div id="heading">
+            <div class="headingImg">
+                <img src="img/logos/facebook.png" class='fb' alt="Facebook logo image">
+                <img src="img/logos/twitter.png" class="twt" alt="twitter logo image">
+            </div>
+            <ul class="headingList">
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </div>
+        <nav class="mobileMenu">
+            <img src="img/buttons/mobile_menu.png" id="abtnImg" alt="Hamburger type button to display menu">
+            <a href="index.html">
+                <div class="logo hide">
+                    <img src="img/logos/icon.png" alt="3d Logo">
+                    <p class="logoText">Africa Green Energy<br> Summit 2022</p>
+                </div>
+            </a>
+            <ul class="navList hide">
+                <li><a href="index.html" class="listAnchor">Home</a></li>
+                <li><a href="index.html" class="listAnchor">Program</a></li>
+                <li><a href="index.html" class="listAnchor">Join</a></li>
+                <li><a href="index.html" class="listAnchor">Sponsor</a></li>
+                <li><a href="index.html" class="listAnchor">News</a></li>
+                <li><a href="index.html" class="listAnchor">Go Green</a></li>
+            </ul>
+        </nav>
+        <div class='abouttxtContainer'>
+            <h1 class="txtGreeting hideAll ">"GO Green today"</h1>
+            <div>
+                <h2 class='atextTitle'>Africa Green Energy Summit 2022</h2>
+            </div>
+            <div class="atxtParagraph">
+                <p class="atxtParagraphp hideAll">Lorem ipsum dolor sit amet consectetur adipisicing elit. In omnis
+                    magnam debitis quam aliquid
+                    illo illum tempora amet sapiente, libero quod quisquam quis ipsam repellat laboriosam totam vel
+                    maxime dolore.
+                </p>
```

Comment:
- [ ] Please use an indentation of two spaces for readability in all your documents. This is a convention and also [best practice.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)